src/gui/libcloudproviders/libcloudproviders.cpp

```diff
@@ -126,7 +126,10 @@ void LibCloudProvidersPrivate::exportFolder(Folder *folder)
         g_error_free(error);
     }
 
-    auto exporter = cloud_providers_account_exporter_new(_exporter, folder->id().constData());
+    // DBus object paths must not contain characters other than [A-Z][a-z][0-9]_, see g_variant_is_object_path
+    const QByteArray dBusCompatibleFolderId = folder->id().replace("-", "").constData();
```

Comment:
Why the constData

Reply:
Was done like this before, too... probably won't hurt.

Reply:
Fixed. For the record, `replace` works inplace. Should we ever change the return type of `folder->id()`, the new snippet should ensure we won't run into problems.